
Mon, 21 Mar 2016 13:04:26 GMT

COMMAND LINE:
/usr/bin/pypy pidigits.pypy-3.pypy 2000

PROGRAM FAILED 


PROGRAM OUTPUT:

Traceback (most recent call last):
  File "pidigits.pypy-3.pypy", line 10, in <module>
    from gmpy import mpz
ImportError: No module named gmpy
